cli/mender-artifact/main.go

```diff
@@ -204,7 +204,7 @@ func getCliContext() *cli.App {
 	writeModuleCommand.Flags = []cli.Flag{
 		cli.StringSliceFlag{
 			Name: "device-type, t",
-			Usage: "Type of device(s) supported by the Artifact. You can specify multiple " +
+			Usage: "[Required] Type of device(s) supported by the Artifact. You can specify multiple " +
```

Comment:
I believe there is a `Required` property in the flag struct that will check that a value is present after parsing.

Reply:
I couldn't find it in the  version of the cli we're using. Did I miss it?

I did double check it now also

Reply:
If we are adding a `[Required]` tag here we should be consistent, and there are several other arguments which are also required. You should double check the list by manual checking, but I know at least `--type` for module-image and `--file` for rootfs-image are required, and `--name` is required for both of them.

Reply:
I see, might as well standardize this then.

This means I'll update the `urfave/cli` dependency also.